jira7/tests/tests.bal

```diff
@@ -421,13 +420,35 @@ function test_createIssue() {
 
     var output = jiraConnectorEP->createIssue(newIssue);
     match output {
-        Issue issue => {
-            issue_test = issue;
-        }
+        Issue issue => issue_test = issue;
 	JiraConnectorError e => test:assertFail(msg = formatJiraConnError(e));
     }
 }
 
+@test:Config {
+    dependsOn: ["test_getProject", "test_getAllIssueTypeStatusesOfProject"]
+}
+function test_createIssueWithExtraFields() {
+    string msg = "";
+    log:printInfo("ACTION : createIssueWithExtraFields()");
+
+    IssueRequest newIssue = {
+        summary: "This is a test issue created for Ballerina Jira Connector with some extra data",
+        issueTypeId: "10004",
+        projectId: project_test.id,
+        assigneeName: config:getAsString("test_username"),
+        description: "test description"
+    };
+    newIssue.reporter = <json>{name: config:getAsString("test_username")};
+    
+    // This should fail to create issue as testField1 and testField2 do not exist on the Jira Project create form
+    var output = jiraConnectorEP->createIssue(newIssue);
```

Comment:
In this test case, we are only adding reporter field and AFAIK it is a standard field in jira, will this result in error? Further the comment states that we are creating issue with the fields "testField1" and "testField2", but seems that is not the case. Could you please check this?

Reply:
We are adding the description and reporter fields. One simple string and one in json format. I have made the test clearer in the commit below. Also changed the small formatting issue highlighted.